ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -335,17 +380,30 @@ azure_file_system_terminate(WT_FILE_SYSTEM *file_system, WT_SESSION *session)
     return 0;
 }
 
+// Checks if a file exists in the container.
 static int
-azure_file_exists(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, bool *existp)
+azure_file_system_exists(
+  WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, bool *existp)
 {
     WT_UNUSED(file_system);
     WT_UNUSED(session);
-    WT_UNUSED(name);
-    WT_UNUSED(existp);
+    azure_file_system *azure_fs = reinterpret_cast<azure_file_system *>(file_system);
 
+    std::cout << "azure_file_system_exists: Checking object: " << name << " exists in Azure."
```

Comment:
Looks like some debugging code slipped through, here and at the end of the function.

Reply:
this is more for explaining the process